docs/GettingStarted/build.md

```diff
@@ -51,15 +106,27 @@ unzip ghidra.zip
 mv ghidra_11.1.1_PUBLIC ~/ghidra
 ```
 
-# Installing Ghidra Scripts
+## Old Stuff
```

Comment:
We should remove old stuff since it may no-longer work and we don't need to maintain it.

Reply:
Sounds good, easy update to make!